code/datums/voice_of_god_command.dm

```diff
@@ -72,7 +72,16 @@ GLOBAL_LIST_INIT(voice_of_god_commands, init_voice_of_god_commands())
 		to_remove_string = "(\\L|^)([to_remove_string])(\\L|$)"
 		message = replacetext(message, regex(to_remove_string, "i"), "")
 
-	var/power_multiplier = base_multiplier * (user.mind?.assigned_role.voice_of_god_power || 1)
+	var/power_multiplier = base_multiplier //strength of user's voice, modified by job
+
+	if (user.mind.assigned_role.departments_bitflags & DEPARTMENT_BITFLAG_COMMAND) //Command staff has authority
+		power_multiplier *= 1.4
+
+	if (user.mind.holy_role)
+		power_multiplier *= 2 //Chaplains are used to speaking for their gods
+
+	if (is_mime_job(user.mind.assigned_role)) //why are you talking
```

Comment:
No no, I want `voice_of_god_power` to exist for stuff like mimes, I just don't want to require it be set for every command.